static/play/play.js

```diff
@@ -122,6 +122,27 @@ addEventListener("DOMContentLoaded", (event) => {
           endLineNumber: error.token?.end.line,
           endColumn: error.token?.end.column,
         })
+      } else if (typeof AggregateError === 'function' && error instanceof AggregateError) {
```

Comment:
The KDL parser throws an `AggregateError`. If that error class is not available (i.e. someone's on IE or old Edge or iOS/Safari <14) that might in itself cause a ReferenceError to be thrown. In this `catch` block, a lack of `ReferenceError` stops the playground from working altogether as soon as a non-InvalidKdlError value is thrown as the instanceof check would cause another `ReferenceError`.

I'm not sure if we consider these older browsers, so I'm happy to toss out the typeof check if you don't like it.